src/burn/drv/neogeo/d_neogeo.cpp

```diff
@@ -1218,7 +1218,7 @@ struct BurnDriver BurnDrvNeoGeoMVS = {
 	"neogeo", NULL, NULL, NULL, "1990",
 	"Neo Geo MVS system\0", "System - cartridges selected seperately", "SNK", "Neo Geo MVS",
 	NULL, NULL, NULL, NULL,
-	BDF_GAME_WORKING /*| BDF_SYSTEM_ONLY*/, 2, HARDWARE_SNK_MVS, GBF_MISC, 0,
+	BDF_GAME_WORKING /*| BDF_SYSTEM_ONLY*/, 2, HARDWARE_SNK_MVS, GBF_BIOS, 0,
 	NULL, neoMVSRomInfo, neoMVSRomName, NULL, NULL, NULL, NULL, neoMVSInputInfo, neoMVSDIPInfo,
```

Comment:
Thanks for all the nice work!
Regarding this one - there was a reason that this was left marked as GBF_MISC.  I can't remember exactly why (someone explained it to me once, it might come back to me later though).  Can you please revert this in your pull request for now?
best regards,
- dink

Reply:
Hey, I've somehow managed to overwrite a commit I didn't want to. I'm still getting used to Github. Please don't accept yet!

Reply:
From a shell, the command "git rebase" allows you to do pretty much anything.

Reply:
Take you're time :)

Reply:
Panic over. Man that was stressful. All sorted.

I didn't even know you could use Git with a shell..... Haha. I'll check it out.

Thanks as usual guys!

Reply:
I do all my development under mingw's bash shell, using fte as an editor (w/tabstops set to 4 and a few other hacks to the config for niceness)
If you're comfortable with cmdline/textmode stuff, its certainly the way to go.

Reply:
Interesting.. I've only really used Visual Studio, Godot, etc. Basic game dev tools. I'm really new to the software side of things. The closest I've come to something like this is making a few mobile apps, c++ games and messing around with hex editors.

I like that MinGW345, Perl and Notepad++ are only about 200mb. I've been using my tablet a lot recently. It's a perfect little arcade machine and I've been making a lot of changes on the fly without having to jump back on my PC. It makes a change having to jump between Photoshop, 3DS Max, Visual Studio, and a million other things.